docs/linux_dependencies.md

```diff
@@ -16,8 +16,8 @@ sudo apt-get install clang
 
 ### Windows Subsystem for Linux (WSL 2)
 
-Graphics and audio need to be configured for them to work with WSL 2 backend. 
-Please see the ubuntu [WSL documentation](https://wiki.ubuntu.com/WSL) on how to set up graphics and audio. 
+Graphics and audio need to be configured for them to work with WSL 2 backend.
+Please see the ubuntu [WSL documentation](https://wiki.ubuntu.com/WSL) on how to set up graphics and audio.
```

Comment:
This part of diff was unintended - I have my editor configured so it removes all the trailing white spaces so I can make myself look more organized to my peers than I actually am. I can revert/reset these lines if you wish.

Reply:
You can propebly keep it, as #1463 also removes all trailing spaces.